app/models/support_interface/provider_user_form.rb

```diff
@@ -66,7 +66,11 @@ def provider_permissions=(attributes)
           provider_id: form.provider_permission[:provider_id],
           provider_user_id: provider_user.try(:id),
         )
-        permission.manage_users = form.provider_permission.fetch(:manage_users, false)
+
+        ProviderPermissions::VALID_PERMISSIONS.each do |permission_name|
+          permission.send("#{permission_name}=", form.provider_permission.fetch(permission_name, false))
```

Comment:
I couldn't decide at first, but I think I like this. :)

Reply:
Metaprogramming considered harmful! 
I wasn't sure about it either, but this seems like a valid use as it simplifies adding further permissions using the `ProviderPermissions::VALID_PERMISSIONS` constant.